ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,61 +13,75 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class="has-top-margin-xl">
-  <p class="is-label">Which secrets would you like us to sync?</p>
-  <p class="sub-text">
-    Select a KV engine mount and path to sync a secret to the destination.
-  </p>
-
+<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
```

Comment:
nit: could we instead add this margin to the error and alert components?

Reply:
The `MessageError` component doesn't spread attributes and I didn't want to wrap it in an extra div. I considered updating `MessageError` since it has fixed margins on it that cannot be overridden which isn't very flexible but it seemed out of scope.